src/components/FormComponents/SourceWizardSummary.js

```diff
@@ -27,22 +28,44 @@ import {
 import { shallowEqual } from 'react-intl/src/utils';
 import { useSelector } from 'react-redux';
 
+const CUSTOMIZING_CUR_LINK = undefined; // specify when docs link is available
+
 const SummaryAlert = ({ appName, sourceType, hcsEnrolled }) => {
   const intl = useIntl();
 
   if (appName === COST_MANAGEMENT_APP_NAME && !hcsEnrolled) {
     return (
-      <Alert
-        variant="info"
-        isInline
-        title={intl.formatMessage({ id: 'cost.rbacWarningTitle', defaultMessage: 'Manage permissions in User Access' })}
-      >
-        {intl.formatMessage({
-          id: 'cost.rbacWarningDescription',
-          defaultMessage:
-            'Make sure to manage permissions for this source in custom roles that contain permissions for Cost Management.',
-        })}
-      </Alert>
+      <Fragment>
+        <Alert
+          variant="info"
+          isInline
+          title={intl.formatMessage({
+            id: 'cost.warningTitleCUR',
+            defaultMessage: "Don't forget additional configuration steps!",
```

Comment:
Let's use single quotes
```suggestion
            defaultMessage: 'Don't forget additional configuration steps!',
```

Reply:
Eslint enforces replacing `'Don\'t forget additional configuration steps!'` with the double quotes.